### DISCUSSION TITLE
How to add to ‘Other Go Recourses\ Video Channels’ section?

### DISCUSSION THREAD

**Discussion by Nemurius** [online-go/online-go.com#2091] [Q&A] [answered]
Hello!
 
Is it possible to add a video channel to the list of Go resources (Other Go Recourses\ Video Channels’ section)?
 
Vladislav Vinokurov, 4D Fox, Russian Champ U12, streams on twitch every Saturday (Russian language).
https://www.twitch.tv/vladislav_vinokurov
https://www.youtube.com/@user-qu1zq7ny5w

Thank you,
Dmitry

**Comment by BHydden** [comment, marked as answer]
Hi Dmitry, 

Thanks for the question!
If you would like to make the change yourself, you could edit [this section](https://github.com/online-go/online-go.com/blob/32027c342a3030e236efc8fbb5373948a395f123/src/views/docs/GoResources.tsx#L407-L648) of the code to add the new channel in the same style, and then create a pull request for anoek to review and merge into the main code.

If you would prefer not to do the coding side of it yourself, I could make those changes for you.

Hope you find this answer helpful,
Let me know how you want to proceed.

- BHydden

**Reply by Nemurius** [reply]
Hi BHydden,
Can you please do it for me? I suppose it should look something like that:

    <span>
                                <Flag country={ru} />{" "}
                                <a rel="noopener" href="https://www.twitch.tv/vladislav_vinokurov">
                                    Vladislav Vinokurov Twitch
                                </a>{" "}
                                /{" "}
                                <a rel="noopener" href="https://www.youtube.com/@vladislav_vinokurov">
                                    YouTube
                                </a>
                            </span>

**Reply by BHydden** [reply]
Yep that's pretty much it 👍 
I've requested the changes for you (#2092), there may be some delay of days or maybe weeks until it is actually merged onto the main site.

**Reply by Nemurius** [reply]
> Hi BHydden, Can you please do it for me? I suppose it should look something like that:
> 
> ```
> <span>
>                             <Flag country={ru} />{" "}
>                             <a rel="noopener" href="https://www.twitch.tv/vladislav_vinokurov">
>                                 Vladislav Vinokurov Twitch
>                             </a>{" "}
>                             /{" "}
>                             <a rel="noopener" href="https://www.youtube.com/@vladislav_vinokurov">
>                                 YouTube
>                             </a>
>                         </span>
> ```

Hello! Thank you once again! Can you change the Youtube link, like in my example? https://www.youtube.com/@vladislav_vinokurov
Seems like old link is no longer working.

**Reply by Nemurius** [reply]
> Yep that's pretty much it 👍 I've requested the changes for you (#2092), there may be some delay of days or maybe weeks until it is actually merged onto the main site.

Hello! Thank you once again! Can you change the Youtube link, like in my example? https://www.youtube.com/@vladislav_vinokurov
Seems like old link is no longer working.

**Reply by BHydden** [reply]
I've requested the changes for you (#2116) 🙂 
thanks for letting me know the other link was broken.
